src/Workspaces/Core/Portable/Storage/SQLite/Interop/SafeSqliteBlobHandle.cs

```diff
@@ -3,37 +3,33 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Runtime.InteropServices;
 using Microsoft.CodeAnalysis.Shared.Extensions;
 using SQLitePCL;
 
 namespace Microsoft.CodeAnalysis.SQLite.Interop
 {
-    internal sealed class SafeSqliteBlobHandle : SafeHandle
+    internal sealed class SafeSqliteBlobHandle : IDisposable
```

Comment:
@sharwell  was not sure how to handle converting these over. the main issue was that the sqlite core types (like sqlite3_blob) are now SafeHandles themselves, and do not expose the `.ptr` value.

Reply:
i thnik we still want these as your leasing pattern effectively means: as long as we create subhandles (i.e. blobs/statements, they should hold the outer sqlite3 instance alive).